src/components/features/ClientFeedback/ClientFeedback.module.scss

```diff
@@ -0,0 +1,69 @@
+@import "../../../styles/settings.scss";
+
+.root {
+  .panelBar {
+    margin-bottom: 30px;
+    position: relative;
+
+    :global(.row) > * {
+      border-bottom: 4px solid #e2e2e2;
+    }
+  }
+
+  .heading {
+    position: relative;
+
+    h3 {
+      color: $primary;
+      text-transform: uppercase;
+      font-size: 20px;
+      line-height: 38px;
+      margin: 0;
+      letter-spacing: 1px;
+    }
+
+    &::before {
+      content: "";
+      position: absolute;
+      bottom: -4px;
+      left: 0;
+      width: 45px;
+      border-bottom: 4px solid $primary;
+    }
+  }
+
+  .dots {
+    position: absolute;
+    right: 0;
+    top: 0;
+
+    ul {
+      margin: 0;
+      padding: 0 0 0 1rem;
+      list-style: none;
+      background-color: #ffffff;
```

Comment:
używamy chyba zmiennych do kolorów, biały to wiem, że na pewno jest bo sama go określałam:) znajdziesz nazwy zmiennych w settings.scss, resztę kolorów też wydaje mi się że trzeba zmienić na zmienne